basics-assignments-4-start/basics-assignments-4-start/src/app/my-inner/my-inner.component.ts

```diff
@@ -0,0 +1,57 @@
+// my-inner.component.ts
+import { Component, EventEmitter, Output} from '@angular/core';
+
+@Component({
+  selector: 'app-my-inner',
+  template: `
+    <div class="mat-container mat-mt-3">
```

Comment:
You don't need to duplicate the same html code here. Write it only in html and add here only templateURL.